src/client/components/app/Transactions/components/TxTokenInput.tsx

```diff
@@ -151,11 +164,9 @@ const scaleTransitionTime = 300;
 
 const StyledTxTokenInput = styled(TransitionGroup)`
   display: grid;
-  background: ${({ theme }) => theme.colors.txModalColors.backgroundVariant};
-  min-height: 15.6rem;
+  // min-height: 15.6rem;
```

Comment:
Remove?

Reply:
I would leave it for another pr too and just focus on technical stuff on this one